tcms/testruns/views.py

```diff
@@ -609,6 +609,7 @@ def cc(request, run_id):  # pylint: disable=invalid-name
 
 
 @require_POST
+@permission_required('testruns.change_testcaserun')
```

Comment:
You should see a failure in this test b/c line:

```
test_case_runs = test_case_runs.filter(status__name='IDLE')
```

will not work reliably with translations.